### ISSUE TITLE
Bug on Profile Container under current pregnancy

### ISSUE THREAD

**Issue by hmakandi** [OpenSRP/opensrp-client-anc#377]
Steps: Go to the Profile Container under current pregnancy, fill in Yes under LMP Known question, and select a date. Note the calculation of the GA. Tap the date to change the date to a significantly different date from the previous one. The GA calculation does not change. It only changes when you go to the next form or the previous form.

- [ ] Is this the expected behaviour?

- [ ] Is it possible to recalculate GA on change?

**Comment by cafootitt** [comment]
Yes, it would be nice for it to recalculate GA on change. Thanks Hildah!
